MIGRATING.md

```diff
@@ -0,0 +1,305 @@
+| `OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT` | Maximum allowed span attribute count | unlimited |
+| `OTEL_SPAN_EVENT_COUNT_LIMIT` | Maximum allowed span event count | unlimited |
+| `OTEL_SPAN_LINK_COUNT_LIMIT` | Maximum allowed span link count | 1000 |
+| `OTEL_EVENT_ATTRIBUTE_COUNT_LIMIT` | Maximum allowed attribute per span event count | unlimited |
+| `OTEL_LINK_ATTRIBUTE_COUNT_LIMIT`| Maximum allowed attribute per span link count | unlimited |
+
+Note: Prior use of [`tracing.WithRecordedValueMaxLength`] should be replaced by
+setting `OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT` to the same value.
+
+### Replace all Manual Instrumentation
+
+All spans created with the [`tracer`] package need to be recreated with
+OpenTelemetry. OpenTelemetry uses `Tracer`s to encapsulate the tracing function
+of a single instrumentation library. Create a `Tracer` from the global
+`TracerProvider` registered when you started the [`distro.SDK`]. To do this use
+the [`otel.Tracer`] function and pass the name of your application.
+
+```go
+tracer := otel.Tracer("my-application")
+```
+
+Use this created `tracer` and its `Start` function to replace all
+[`tracer.StartSpan`] invocations. The `operationName` parameter can be used as
+the `name` parameter for `Start`. The following replacements are used for
+[`tracer.StartSpanOption`] instances.
+
+| [`tracer.StartSpanOption`] | Replacement |
+| --- | --- |
+| [`tracer.ChildOf`] | N/A (see below) |
+| [`tracer.ResourceName`] | N/A (see below) |
+| [`tracer.ServiceName`] | N/A (see below) |
+| [`tracer.SpanType`] | [`trace.WithSpanKind`] |
+| [`tracer.StartTime`] | [`trace.WithTimestamp`] |
+| [`tracer.Tag`] | [`trace.WithAttributes`] |
+| [`tracer.WithRecordedValueMaxLength`] | N/A (see below) |
+| [`tracer.WithSpanID`] | N/A (see below) |
+
+Notes:
+
+- [`tracer.ChildOf`]: The parent-child relationship of spans is defined with a
+  [`context.Context`]. The [`context.Context`] passed to `Start` needs to
+  contain the parent span. This is automatically done if the context was
+  returned from a previous call to `Start`. It can explicitly be done using the
+  [`trace.ContextWithSpan`].
+- [`tracer.ResourceName`]: This value is defined global with a [`Resource`]
+  that applies to all spans. See [Defining a Resource](#defining-a-resource)
+  for more information.
+- [`tracer.ServiceName`]: This value is defined global in a [`Resource`] that
+  applies to all spans. See [Defining a Resource](#defining-a-resource) for
+  more information.
+- [`tracer.WithRecordedValueMaxLength`]: This value is set globally. See
+  [Setting Span Limits](#setting-span-limits) for more information.
+- [`tracer.WithSpanID`]: Span IDs are automatically set. If custom span IDs are
+  needed you will need to create a custom [`IDGenerator`].
+
+Finally, the created span, similar to before, needs to be ended. Use the
+OpenTelemetry span's `End` method to do this.
+
+### Replace all Instrumentation Libraries
+
```

Comment:
```suggestion
Replace the following tracing libraries with their OpenTelemetry equivalent:
```

Same question here about the link. It's a great deal of links to packages; the name of the packages wouldn't suffice?